src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        <span>{' by '}</span>
```

Comment:
It's a bad practice for adding spaces this way, it's better to use margins and paddings for that

Reply:
You mean to remove the spaces inside the { } and add them using css for the teg <span>. Did I understand correctly?